resources/views/bread/browse.blade.php

```diff
@@ -223,13 +222,14 @@
         <div class="modal-dialog">
             <div class="modal-content">
                 <div class="modal-header">
-                    <button type="button" class="close" data-dismiss="modal" aria-label="{{ __('voyager::generic.close') }}"><span
-                                aria-hidden="true">&times;</span></button>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="{{ __('voyager::generic.close') }}">
+                        <span aria-hidden="true">&times;</span>
```

Comment:
This (and the couple others like it) are probably fine to be on one line.  I believe whitespace within a `<button>` element is respected, so it surrounding the span could potentially cause issues, primarily in older browsers.  There's no certainty that it will, but I'm okay with following that pattern.